model/src/main/kotlin/utils/OrtResultExtensions.kt

```diff
@@ -88,5 +88,5 @@ fun OrtResult.getDetectedLicensesWithCopyrights(
 /**
  * Copy this [OrtResult] and add all [labels] to the existing labels, overwriting existing labels on conflict.
  */
-fun OrtResult.mergeLabels(labels: Map<String, String>) =
-    copy(labels = this.labels + labels).apply { data += this@mergeLabels.data }
+fun OrtResult.mergeLabels(labels: Collection<String>) =
+    copy(labels = (this.labels + labels).toSortedSet()).apply { data += this@mergeLabels.data }
```

Comment:
Just more thinking out aloud: If the field dos not need to be mutable ans was `val labels: Set<String> = emptySet()`, then this could be `labels = this.labels + labels.toSet()` which arguable is a tad nicer.